src/Extractor.cc

```diff
@@ -2598,6 +2651,19 @@ namespace insur {
             pconverter << myRingInfo.radiusMid;
             alg.parameters.push_back(numericParam(xml_radius, pconverter.str()));
             pconverter.str("");
+            if(myRingInfo.discNumber <=8 && (phi_two[ringIndex]).size()>0 ){
+              alg.parameters.push_back(numericParam("hasExtraInfo", "1"));
```

Comment:
You should not use this algorithm anyway, but just for consistency I would mention:
Please avoid calling variables **hasExtraInfo**, because after other developments are made, one forgets what the extra info is. This would be confusing if the previous algo arguments were called 'NewDev', 'isShiftedGeo', etc.
Here the idea is **isRingRegular** or similar.
But indeed not important at prototyping phase.